src/Footer/style.scss

```diff
@@ -0,0 +1,72 @@
+@import '../variables/_variables.scss';
+
+.info {
+  display: flex;
+  padding: 50px 60px 70px;
+  text-align: center;
+  background: fixed url('../image/background-foo.png') no-repeat center center / cover;
+
+
+  @media screen and (max-width: $max-sm-screen) {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    margin-left: 0;
+    text-align: center;
+}
+
+  h1 {
+    margin-bottom: 20px;
+    font-size: 21px;
+    font-weight: bold;
+    color: $white;
+  }
+
+  p {
+    margin-top: 20px;
+    font-size: 1rem;
+    line-height: 1.5;
+    color: $white;
+
+  }
+
+  span {
+    font-weight: bold;
+  }
+}
+
+.About-footer {
+  margin-left: 25%;
+}
+
+.open {
+  margin-left: 100px;
+}
+
+.Our-Location {
+  margin-left: 100px;
+
+  img {
+    margin-top: 10px;
+  }
+
+  h1 {
+    margin-bottom: 10px;
+  }
+}
+
+
+div {
```

Comment:
for all divs in the project?

Reply:
i cheked that item so i  have removed it